### DISCUSSION TITLE
WixSharp for WixTolset4.

### DISCUSSION THREAD

**Discussion by gillesnoyer** [oleg-shilo/wixsharp#1452] [General]
Hi Everybody 
I use WixSharp to build my project, it's a great tool. The current version is 1.24.2 and I read on some articles that Wix# will be compatible with WixToolSet 4. I tried to use it, but I couldn't find the nuget package, where is it possible to find it? and do you know the current status of this version
THANKS

**Comment by oleg-shilo** [comment]
https://www.nuget.org/packages/WixSharp_wix4

However, the best way is to install WixSharp VS project templates extension and then the whole project setup (including packages) will be done for you when you create your project.

**Comment by gillesnoyer** [comment]
Great thank you, by using NuGet\Install-Package WixSharp_wix4 -Version 2.0.5 I get the package.
